_pages/e10s-addons.md

```diff
@@ -1,15 +1,21 @@
 ---
-layout: page
+layout: activity
 title:  "e10s and Featured Add-ons"
+subtitle: "In like no time because it's retired"
```

Comment:
I'd remove the subtitle completely for retired ones. Also, can we add a light red background to the text or something similar to better show that this is retired?